src/astro/constants.py

```diff
@@ -0,0 +1 @@
+DEFAULT_CHUNK_SIZE = 2000
```

Comment:
Change this to 1,000,000 based on the numbers:
https://github.com/astro-projects/astro/pull/105#discussion_r806605527

Reply:
@tatiana why does this need to be changed in a separate PR?